core/examples/luigi-sample-angular/src/app/project/project.component.html

```diff
@@ -53,6 +53,23 @@ <h1 class="fd-section__title">Project {{ projectId }}</h1>
         <a href="javascript:void(0)" (click)="luigiClient.linkManager().navigate('/settings', null, true)">with preserved view: project to global settings and back</a>
         <app-code-snippet data="luigiClient.linkManager().navigate('/settings', null, true)"></app-code-snippet>
       </li>
+      <li class="fd-list-group__item check-path">
+        <a href="javascript:void(0)" (click)="luigiClient.linkManager().navigate('/settings', null, true)">Check if path
```

Comment:
Please remove the click handler since it navigates to settings.

Reply:
Good catch, I forgot to do that before creating the pr.